README.md

```diff
@@ -2,21 +2,131 @@
+   - The PR(s) scheduled for "**Reading**" will be noted in the teleconference agenda sent out before the meeting.
+   - A "**Straw Poll**" comment is placed on the PR once the "**Reading**" announcement email has been sent out by the author(s).
+     - The "**Straw Poll**" provides the community with a sense of how many people have viewed the PR and are following the discussion. This serves no official purpose other than to help define participation in the consensus building process.
+     - Those participants that approve the "**Reading**" must indicate that by placing a :+1: emoji on the comment.
+     - Those participants that have specific concerns regarding the "**Reading**" must indicate that by placing a :-1: emoji on the comment.
+         - Anyone indicating concern must provide a comment on the ticket following the "**Staw Poll**" comment. The author(s) can then work to address the concern.
+     - Participation in the "**Straw Poll**" starts when the comment is posted and concludes after the teleconference where the PR is presented for "**Reading**"
+
+
+### Phase 3: GitHub PR "Reading"
+
+ * One of the author(s) (or someone designated by them) must be present on the teleconference to present the PR for "**Reading**" and discussion.
+ * In the teleconference, the presenter will discuss the PR and address any questions raised.
+   - During the presentation someone will be designated to take notes on the PR (and/or Issue) for the presenter.
+   - These notes will declare if the PR is eligible for "**Acceptance**" in the next teleconference.
+     - The PR is eligible for "**Acceptance**" if no revisions are required and all objections have been addressed.
+       Any revisions or objections raised during the teleconference will be noted on the PR during or immediately after the meeting.
+ * The PR may be voluntarily closed as "**Withdrawn**" by the corresponding author(s) for any reason. For example, if a dissenting opinion is raised that cannot be resolved in the scope of that PR.
+ * If revisions are required or further discussion is needed then the interested parties should continue discussion on the PR.
+   - If no changes to the PR result from the discussion then the PR can be considered for "**Acceptance**" without the need for another "**Reading**".
+   - If changes are made to the PR (with reasonable exceptions for minor spelling/grammar changes) then another "**Reading**" must be scheduled (See Phase 2) to allow interested parties to review the changes. This will include another "**Straw Poll**".
+   - If major changes are required to the PR the author(s) should consider a separate, replacement PR, and marking the original PR as "**Withdrawn**".
+ * If no dissent is voiced then the PR can be considered for "**Acceptance**" as early as the next teleconference.
```

Comment:
This requirement is too stringent. This can be understood as veto power, which would make any progress impossible.

Reply:
Maybe we can reword to:
> If all dissenting opinions and questions have been addressed then the PR can be considered for "**Acceptance**" as early as the next teleconference.

Leaving the definition of `addressed` open. We will need to more crisply define that.